artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnector.java

```diff
@@ -416,6 +404,13 @@ private String getHttpUpgradeInfo() {
       return ", activemqServerName=" + serverName + ", httpUpgradeEndpoint=" + acceptor;
    }
 
+   private String realKeyStorePath;
```

Comment:
These should remain within method scope.

Reply:
If I move the variables into the method then I have to add some new copy variables because they have to be final or effective final. Is it ok?